_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: seo drupal drupal8
+summary: SEO trends 2017 and Drupal 8 as the best option for your long-term marketing campaign.
+featured_image: "/assets/img/blog/supercharged-seo-with-drupal-8.jpg"
+featured_image_alt: "Supercharged SEO with Drupal 8"
+---
+
+Here at Savas Labs, when it comes to developing systems, our focus is not only to code according to best practices, but also deliver a solid marketing vehicle to our clients and partners. Let’s be honest here - perfectly coded, a pretty looking website will be of little use if it doesn’t produce leads / facilitate conversions or generate revenue! So how do we achieve those goals? We get there by balancing website objectives while giving priority to lead generation via SEO. The more quality traffic that comes to the website - the more conversions we can achieve. It’s that simple!
+
+There are multiple ways of generating traffic to a website. The most popular are SEO (search engine optimization) and PPC (Pay Per Click). Both bring traffic through search engines. The difference between the two is that SEO brings long-term results boosting organic traffic while PPC helps marketers achieve short-term goals by gaining instant exposure throughout the duration of the ad campaign.
```

Comment:
SEO and PPC are both capitalized but when you spell them out in the parentheses there is not consistent capitalization. Please stick to one.